README.md

```diff
@@ -2,17 +2,17 @@
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+## Book store
```

Comment:
- [ ] Please kindly update your `readme file` to reflect the changes for this milestone, checkout this [template](https://github.com/microverseinc/readme-template) and adapt it to your project as this will make your `readme file` more informative and professional.